platform/ui/src/components/FilterDateRange/FilterDateRange.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { DateRange, FilterWrapper } from '@ohif/ui';
+
+const FilterDateRange = ({
+  label,
+  isSortable,
+  isBeingSorted,
+  sortDirection,
+  onLabelClick,
```

Comment:
Should this be changed to `onSortUpdate` or something similar?

Reply:
Maybe `onLabelClick` does make more sense 🤔 
LMK your thoughts.

Reply:
I have added LabelClick because used can do anything he wanted in case any click on label accours.